plugins/modules/cloud/scaleway/scaleway_container_registry_info.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Serverless container registry info module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_container_registry_info
+short_description: Scaleway Container registry info module
+version_added: 5.8.0
+author: Guillaume MARTINEZ (@Lunik)
+description:
+  - This module return info on container registry on Scaleway account.
+extends_documentation_fragment:
+  - community.general.scaleway
+
+
+options:
+  project_id:
+    type: str
+    description:
+      - Project identifier.
+    required: true
+
+  region:
+    type: str
+    description:
+      - Scaleway region to use (for example C(fr-par)).
+    required: true
+    choices:
+      - fr-par
+      - nl-ams
+      - pl-waw
+
+  name:
+    type: str
+    description:
+      - Name of the container registry.
+    required: true
+'''
+
+EXAMPLES = '''
+- name: Get a container registry info
+  community.general.scaleway_container_registry_info:
+    project_id: '{{ scw_project }}'
+    region: fr-par
+    name: my-awesome-container-registry
+  register: container_registry_info_task
+'''
+
+RETURN = '''
+container_registry:
+    description: The container registry informations.
+    returned: always
+    type: dict
+    sample: {
```

Comment:
Same comment as in the other module wrt YAML vs. JSON idioms.

Reply:
Fixed in ac03fd5c5b30d99f508cb2699f9ff3513fabc44a